upgrade-2.x.sh

```diff
@@ -205,18 +210,18 @@ function upgrade_supervisor() {
         fi
     fi
 
-    if CURRENT_SUPERVISOR_VERSION=$(curl --retry 10 --silent --header "Authorization: Bearer ${APIKEY}" "${API_ENDPOINT}/v5/device(${DEVICEID})?\$select=supervisor_version" | jq -r '.d[0].supervisor_version'); then
+    if CURRENT_SUPERVISOR_VERSION=$(CURL_CA_BUNDLE=${TMPCRT} curl --retry 10 --silent --header "Authorization: Bearer ${APIKEY}" "${API_ENDPOINT}/v5/device(${DEVICEID})?\$select=supervisor_version" | jq -r '.d[0].supervisor_version'); then
```

Comment:
Is it worth breaking out the curl commands to a separate command?  Maybe adding something after the setup of the TMPCRT file:

```
CURLB="CURL_CA_BUNDLE=${TMPCRT} curl"
```